contracts/escrow/dapps/AaveClaimAaveFacet.sol

```diff
@@ -53,29 +63,27 @@ contract AaveClaimAaveFacet is PausableMods, DappMods {
         uint256 amount,
         address[] calldata tokenAddresses
     ) public paused("", false) onlyBorrower(loanID) {
-        address escrow = address(LibEscrow.e(loanID));
+        address user = LibLoans.loan(loanID).status >= LoanStatus.Closed
+            ? msg.sender
+            : address(LibEscrow.e(loanID));
         bytes memory result = LibEscrow.e(loanID).callDapp(
             address(INCENTIVES_CONTROLLER_ADDRESS),
             abi.encodeWithSelector(
                 IAaveIncentivesController.claimRewards.selector,
                 tokenAddresses,
                 amount,
-                escrow
+                user
             )
         );
 
         bytes memory unstake = LibEscrow.e(loanID).callDapp(
             address(STAKE_TOKEN_ADDRESS),
-            abi.encodeWithSelector(
-                IStakedAave.redeem.selector,
-                msg.sender,
-                amount
-            )
+            abi.encodeWithSelector(IStakedAave.redeem.selector, user, amount)
         );
 
         // Add AAVE to escrow token list
         // TODO: get the AAVE token (add as immutable constructor variable)
-        // LibEscrow.tokenUpdated(loanID, address(0));
+        LibEscrow.tokenUpdated(loanID, address(0));
```

Comment:
1. this should fail when called as this calls `balanceOf` on the token address passed in. 
2. should only call this if the escrow contract claims the reward (if user's address gets the reward, no need to update the escrow token array)

Reply:
will do